stacks-signer/Cargo.toml

```diff
@@ -42,6 +42,7 @@ toml = "0.5.6"
 tracing = "0.1.37"
 tracing-subscriber = { version = "0.3.17", features = ["env-filter"] }
 wsts = { workspace = true }
+sha2 = "0.10.6"
```

Comment:
We should probably try to use the same version sha2 as used throughout the rest of the stacks core.

I see quite a few use this

```
[target.'cfg(all(any(target_arch = "x86_64", target_arch = "x86", target_arch = "aarch64"), not(target_env = "msvc")))'.dependencies]
sha2 = { version = "0.10", features = ["asm"] }

[target.'cfg(any(not(any(target_arch = "x86_64", target_arch = "x86", target_arch = "aarch64")), target_env = "msvc"))'.dependencies]
sha2 = { version = "0.10" }
````

I think maybe we should add this to the outer Cargo.toml file and then put sha2 = { workspace = true } in all other sub repos that require it. Not 100% sure if this is required but when Jude or Aaron gives a review they can confirm or reject it

Reply:
Makes sense. Moved it to outer Cargo.toml.

Reply:
Sorry. My bad making you reverse this change :(